haystack/nodes/reader/farm.py

```diff
@@ -1114,9 +1114,7 @@ def eval(
         # Create DataLoader that can be passed to the Evaluator
         tic = perf_counter()
         indices = range(len(farm_input))
-        dataset, tensor_names, problematic_ids = self.inferencer.processor.dataset_from_dicts(
-            farm_input, indices=indices
-        )
+        dataset, tensor_names, _ = self.inferencer.processor.dataset_from_dicts(farm_input, indices=indices)
```

Comment:
Just from the name, do you think maybe we should add a debug log of the problematic_ids?

Reply:
Nice idea but I'd not touch this section of the code :sweat_smile: If it was not logged to begin with, it's probably not that important